src/AspNetCoreModuleV2/CommonLib/exceptions.h

```diff
@@ -74,6 +77,26 @@
 #define SUCCEEDED_LOG(hr)                                       SUCCEEDED(LOG_IF_FAILED(hr))
 #define FAILED_LOG(hr)                                          FAILED(LOG_IF_FAILED(hr))
 
+inline thread_local IHttpTraceContext* g_traceContext;
```

Comment:
It's weird to have this here. Can you add this to ModuleTracer. Or at least make:
```
    if (ANCMEvents::ANCM_HRESULT_FAILED::IsEnabled(pTraceContext))
    {
        ANCMEvents::ANCM_HRESULT_FAILED::RaiseEvent(pTraceContext, nullptr, fileName, lineNumber, hr);
    }
```
a static method in ModuleTracer.h